packages/communication-react/review/communication-react.api.md

```diff
@@ -1337,6 +1337,7 @@ export type VideoGalleryParticipant = {
 
 // @public
 export interface VideoGalleryProps {
+    layout?: 'default' | 'floatingLocalVideo';
```

Comment:
I worry about this field not being future proof: If we start adding more layouts, we'll see combinatorial growth in options: `'default' | 'floatingLocalVideoAndFunkyHair' ...`

A more specific field will make this more readable and avoid bloat in the future: 

    /** 
      * Default is docked
      */
    localVideoPosition: 'docked' | 'floating'

Reply:
Thought about that. That means we will need different fields for remote participant layout, screenshare layout and local preview layout with no way of limiting combinations. So contoso can mess up the entire layout pretty easily.

Reply:
Makes sense to only allow combinations that are actually supported.
Can you list the expected values in this enum eventually?

`layout` still sounds quite general (and has a different meaning?).

Reply:
Possible layouts
- SpeakerCentered
- OnlySpeakerVideo
- LocalVideoCentered
- ScreenshareOnly
- ScreenShareCentered

And the list goes on and on...

Curious how `layout` is a general word?
What other meanings are you thinking of when using the term layout with a videogallery ?